### PR TITLE
Update the docs for using the SDK locally and with CE

### PR DESCRIPTION
**Pull request by hidday**
Update the README for using the SDK locally and with the academy chrome extension, both with dev and with prod refrences.

### CODE REVIEW

**Review by DvirMalka — APPROVED** [review]

**Comment by DvirMalka** [comment]
@hidday @guzzur 
should we sync on it?

**Comment by hidday** [comment]
We can

File: README.md [hunk 1]
```diff
@@ -98,6 +98,53 @@ The academy panel will be closed, `sessionStorage` and `localStorage` configurat
...
 # Development
 
+# Local
+For developing with your local services and SDK:
+
+* Add `127.0.0.1       local.strigo.io` to your hosts file (`/etc/hosts`)
+* Set your `IS_DEVELOPMENT` variable to `true` in the `.env` file.
+
+## Working with the Strigo Chrome extension locally
+To allow a smoother dev experience with ability to test you changes in our Chrome extension
```
Comment (guzzur) [inline README.md:108]: Your changes

File: README.md [hunk 2]
```diff
@@ -98,6 +98,53 @@ The academy panel will be closed, `sessionStorage` and `localStorage` configurat
...
+you can direct your local dev server to also write your latest `strigo.sdk.js` bundle 
+into the *strigo-academy-chrome-extension* scripts folder. 
+
+A new build of the extension is not required in that case since Chrome extensions invoke everything in the `scripts` folder dynamically.
+
+The only thing you might need in order tou test your changes is to refresh the page.
+
+-- in case you don't see you changes in the CE - try to update the extension in the `chrome://extensions` page.
```
Comment (guzzur) [inline README.md:116]: Your

File: README.md [hunk 3]
```diff
@@ -98,6 +98,53 @@ The academy panel will be closed, `sessionStorage` and `localStorage` configurat
...
+
+#### To start the dev server with the extension-hot-reload, run:
+```sh
+npm start:ext
```
Comment (guzzur) [inline README.md:120]: Does this work? It is not a native script. I'd suggest `npm run start:ext`

File: README.md [hunk 4]
```diff
@@ -98,6 +98,53 @@ The academy panel will be closed, `sessionStorage` and `localStorage` configurat
...
+
+---
+We are "temporarily" 🙄 using the `dist/development/strigo.sdk.js` build for production as well!
+
+So the actual version you upload as the "prod" version of things should be the development build
+
+but with the **IS_DEVELOPMENT** variable set to **false**!
+____
+
+To build a "development" build with the IS_DEVELOPMENT flag set to false you can either run the `start:extension` command,
```
Comment (guzzur) [inline README.md:136]: Didn't we just said it was `start:ext`?

**Review by guzzur — COMMENTED** [review]
You overrode last PRs with this PR

File: README.md (hunk 3 above)
Comment (hidday) [reply README.md:120]: true

File: README.md (hunk 4 above)
Comment (hidday) [reply README.md:136]: it was.
Then I saw that omry called it "start:extension" so I kept his....

**Pull request merged**